custom_connectors/custom_auth/eventbrite_connector.rb

```diff
@@ -0,0 +1,104 @@
+{
+  title: "Eventbrite (custom)",
+
+  connection: {
+    fields: [
+      {
+        name: "personal_token",
+        control_type: "password",
+        hint: "Find your personal token here: https://www.eventbrite.com/myaccount/apps/",
+        optional: false
+      }
+    ],
+
+    authorization: {
+      type: "custom_auth",
+
+      apply: lambda do |connection|
+        headers("Authorization": "Bearer #{connection["personal_token"]}")
+      end
+    },
+
+    base_uri: lambda do
+      "https://www.eventbriteapi.com"
+    end
+  },
+
+  test: lambda do
+    get("/v3/users/me")
+  end,
+
+  actions: {
+    get_sales_report: {
```

Comment:
This schema is too long. We should choose an object that has fairly flat structure and has limited number of field(< 10).